pkg/conversion/struct_test.go

```diff
@@ -64,3 +64,44 @@ func TestConversion(t *testing.T) {
 		t.Error("StructToMessage(nil) => got no error")
 	}
 }
+
+// BENCHMARKS =====================================================================================================
+
+func BenchmarkConversion(b *testing.B) {
```

Comment:
I think this one should be looping over b.N?

Reply:
```go
func BenchmarkConversion(b *testing.B) {
	for n := 0; n < b.N; n++ {
		pb := &discovery.DiscoveryRequest{
			VersionInfo: "test",
			Node:        &core.Node{Id: "proxy"},
		}
		st, err := conversion.MessageToStruct(pb)
		if err != nil {
			b.Fatalf("unexpected error %v", err)
		}

		out := &discovery.DiscoveryRequest{}
		err = conversion.StructToMessage(st, out)
		if err != nil {
			b.Fatalf("unexpected error %v", err)
		}
	}
}
```
More concise and really more of a benchmark 👍